topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>    {: .comment}
+>
+{: .hands_on}
+
+
+> ### {% icon question %} Questions
+>
+> 1. Can you use this result to say how many sequences occuring only once? You would need to use Galaxy tools like `Sort data in ascending or descending order` and ` Select first lines from a dataset` to answer the question
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. 3131 sequences are occuring once.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Filtering sequances by count and length with **obigrep**
+
+> ### {% icon hands_on %} Hands-on: Keep only the sequences having a count greater or equal to 10 and a length shorter than 80 bp
+>
+> 1. {% tool [obigrep](toolshed.g2.bx.psu.edu/repos/iuc/obi_grep/obi_grep/1.2.13) %} with the following parameters:
+>    - *"Choose the sequence record selection option"*: `predicat`
+>        - *"Python boolean expression to be evaluated for each sequence record."*: `count>=10`
+>
+> 2. {% tool [obigrep](toolshed.g2.bx.psu.edu/repos/iuc/obi_grep/obi_grep/1.2.13) %} with the following parameters:
+>    - *"Choose the sequence record selection option"*: `lmin`
+>        - *"lmin"*: `80`
+>
+>    > ### {% icon comment %} Comment
+>    > Based on the previous observation, we set the cut-off for keeping sequences for further analysis to a count of 10
+>    > Based on previous knowledge we also remove sequences with a length shorter than 80 bp (option -l) as we know that the amplified 12S-V5 barcode for vertebrates must have a length around 100bp
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many sequences are kept following the "count" filter?
+> 2. How many sequences are kept following the "length" filter?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. 178
+> > 2. 175
+> >
+> {: .solution}
+>
+{: .question}
+
+## Clean the sequences for PCR/sequencing errors (sequence variants) with **obiclean**
+
+> ### {% icon hands_on %} Hands-on: Clean the sequences for PCR/sequencing errors (sequence variants)
+>
+> 1. {% tool [obiclean](toolshed.g2.bx.psu.edu/repos/iuc/obi_clean/obi_clean/1.2.13) %} with the following parameters:
+>    - *"Input sequences file"*: `obigrep output file`
+>    - *"Maximum numbers of differences between two variant sequences (default: 1)"*: `1`
+>    -  *"Threshold ratio between counts (rare/abundant counts) of two sequence records so that the less abundant one is a variant of the more abundant (default: 1, i.e. all less abundant sequences are variants)"*: `0.05`
+>    - *"Do you want to select only sequences with the head status in a least one sample?"*: `Yes`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > As a final denoising step, using the obiclean program, we keep the head sequences that are sequences with no variants with a count greater than 5% of their own count
+>    {: .comment}
+>
+{: .hands_on}
+
+
+## Taxonomic assignment of sequences with **NCBI BLAST+ blastn**
+
```

Comment:
Could you explain a little bit why this step is important? And what is the objective?

Reply:
Once again, you're right! THANK YOU VERY MUCH!!!

Reply:
done!